ec2/sc-ec2-linux-ra.yaml

```diff
@@ -12,16 +12,22 @@ Metadata:
           default: Linux Instance Configuration
         Parameters:
           - EC2InstanceType
-          - LatestAmiId
+          - LinuxDistribution
     ParameterLabels:
       KeyPair:
         default: Key Pair
       PrivateNetwork:
         default: Use private network?
       EC2InstanceType:
         default: EC2 Instance Type
-      LatestAmiId:
-        default: SSM key to the latest Amazon linux AMI
+      LinuxDistribution:
+        default: Linux Distribution
+Mappings:
+  LinuxDistributions:
+    Ubuntu:
+      AMIID: ami-04b9e92b5572fa0d1
```

Comment:
If these are AWS AMIs they are not guaranteed to be around.  We should save a copy of these in our account.